### PR TITLE
Feature/basic ui fixes

### PR DESCRIPTION
**Pull request by LisaQ04**
made new changes to UI.

### CODE REVIEW

File: waterlog/src/App.scss [hunk 1]
```diff
@@ -38,7 +38,8 @@ button {
 }
 
 .bold {
-	font-weight: bold;
+  font-weight: bold;
+  color: 	#C0C0C0;
```
Comment (FredJEhlers) [inline waterlog/src/App.scss]: incorrect spacing

File: waterlog/src/Components/DailyCostsReports.js [hunk 2]
```diff
@@ -20,24 +22,24 @@ const DailyCostsReports = (props) => {
     labels: labelX,
     datasets: [
       {
-        label: 'cost',
+        label: 'current cost',
         data: dataY,
         fill: true,
         borderColor: '#56ccf7',
```
Comment (FredJEhlers) [inline waterlog/src/Components/DailyCostsReports.js:28]: You are using these colours a lot make use of constants, please. Same goes for `#C0C0C0`

File: waterlog/src/Components/TankGraph.js [hunk 3]
```diff
@@ -1,44 +1,24 @@
...
-        pointHitRadius: 5
-      },
-      
+    datasets: [ 
       {
         label: 'rands',
-        data: dataY,
+        data:  dataY,
         fill: true,
         borderColor: 'rgba(23,209,255,1)',
```
Comment (FredJEhlers) [inline waterlog/src/Components/TankGraph.js:17]: you guys are mizng rgba and hex choose one and stick to it throughout the project

**Review by FredJEhlers — CHANGES_REQUESTED** [review]

File: waterlog/src/utils.js [hunk 4]
```diff
@@ -42,6 +42,17 @@ export const formatDate = (date) => {
 	return [ month, day, year ].join('/');
 };
 
+export const mapDataTime = (myData) =>
+{
+  let index = [];
+  let dataY = new Array(24).fill(null);;
```
Comment (GTaylorRabbit) [inline waterlog/src/utils.js]: Please see the double ;; in this line

File: waterlog/src/Components/TankGraph.js (hunk 3 above)
Comment (LisaQ04) [reply waterlog/src/Components/TankGraph.js:17]: Thanks its being fixed on this PR: https://github.com/Hunter27/WaterLog-Front-end/pull/106

File: waterlog/src/Components/DailyCostsReports.js (hunk 2 above)
Comment (LisaQ04) [reply waterlog/src/Components/DailyCostsReports.js:28]: Thanks its being fixed: https://github.com/Hunter27/WaterLog-Front-end/pull/106

File: waterlog/src/utils.js (hunk 4 above)
Comment (LisaQ04) [reply waterlog/src/utils.js]: Thanks fixed it.

File: waterlog/src/App.scss (hunk 1 above)
Comment (LisaQ04) [reply waterlog/src/App.scss]: Fixed spacing.

**Review by FredJEhlers — APPROVED** [review]

**Pull request merged**